content/en/docs/tasks/debug-application-cluster/monitoring/monitor-node-health.md

```diff
@@ -4,6 +4,7 @@ content_type: task
 reviewers:
 - Random-Liu
 - dchen1107
+weight: 20
```

Comment:
Maybe put this file and following files under `content/docs/tasks/debug-cluster/`?